src/wrapper/wrapper.c

```diff
@@ -216,15 +158,15 @@ real_call:;
     return (func_ptr (path, oflag, mode));
 }
 
-FILE *fopen (const char *path, const char *mode)
+DYAD_DLL_EXPORTED FILE *fopen (const char *path, const char *mode)
 {
     char *error = NULL;
     typedef FILE *(*fopen_ptr_t) (const char *, const char *);
     fopen_ptr_t func_ptr = NULL;
 
-    if (ctx == NULL) {
-        dyad_wrapper_init ();
-    }
+    // if (ctx == NULL) {
+    //     dyad_wrapper_init ();
+    // }
```

Comment:
Same comment as for open. Probably remove this line.

Reply:
Done